changelog/7782.md

```diff
@@ -8,3 +8,8 @@
     - Django from 4.0 to 4.1
       - apps:  added max_length in multiselectfield to fix index error with
         validators[0]=MaxValueMultiFieldValidator(self.max_length)
+      - apps:  added migration for wagtail custom renditions
+
+    - Django from 4.1 to 4.2
+      - site-packages: update taggit to v4. 
+        Should update to latest version when compatible with wagtail to fix index_together deprecation
```

Comment:
is this only compatible with wagtail 5.2 or why do we need to wait ?

Reply:
yes, current wagtail 5.1.3 compatible to taggit > 5.0